x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/home/index.tsx

```diff
@@ -100,7 +108,7 @@ export const mapToCard = ({
     fromIntegrations: selectedCategory,
     integration: 'integration' in item ? item.integration || '' : '',
     name: 'name' in item ? item.name : item.id,
-    version: 'version' in item ? item.version || '' : '',
+    version,
```

Comment:
Fixing a bug when a GA version is installed (e.g. `endpoint-8.5.0`), the release badge on `Installed integrations` tab was showing `Beta`. I think this was misleading and the badge should show the installed version's release.

Also, on `Browse integrations` tab, changed the integration card to link to the latest version of the integration, not the installed one as currently.